src/locales/en/translation.json

```diff
@@ -75,11 +75,16 @@
   "assetTransfer": "Asset Transfer",
   "functionCall": "Function Call",
   "functionCalls": "Function Calls",
-  "permission": "Permission",
-  "setPermissions": "Set Permissions",
-  "setPermissionsFor": "Set permissions for",
-  "setApprovalPermissionsFor": "Set approval permissions for",
-  "setTransferPermissionsFor": "Set transfer permissions for",
+  "permissions": {
+    "setPermissions": "Set Permissions",
+    "setPermissionsFor": "Set permissions for",
+    "setApprovalPermissionsFor": "Set approval permissions for",
+    "setTransferPermissionsFor": "Set transfer permissions for",
+    "permissionDenied": "Permission denied",
+    "permissionDeniedMessage": "You do not have permission to perform this action.",

```

Comment:
I think this message is more fitting since its not a user permission issue

"The DAO does not have permission to perform this action"